test/unit/Iterate_with_test.lua

```diff
@@ -0,0 +1,26 @@
+local expirationd = require("expirationd")
+local t = require("luatest")
+local g = t.group("iterate_with")
+
+local helpers = require("test.helper")
+
+g.before_all(function()
+    helpers.init_spaces(g)
+end)
+
+g.after_each(function()
+    helpers.truncate_spaces(g)
+end)
+
+function g.test_passing()
+    local task = expirationd.start("clean_all", g.tree.id, helpers.is_expired_true,
+            { iterate_with = helpers.is_expired_true })
+    -- default process_while always return false, iterations never stopped by this function
+    t.assert_equals(task.iterate_with, helpers.is_expired_true)
+    task:kill()
+
+    --errors
```

Comment:
Why do you add a space between `--` and a sentence in some cases but don't do it for `--errors`?

Reply:
Copy paste problem